src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/DispatchOperation.cs

```diff
@@ -188,7 +188,7 @@ public string Name
             get { return name; }
         }
 
-        internal SynchronizedCollection<IParameterInspector> ParameterInspectors { get; }
+        public SynchronizedCollection<IParameterInspector> ParameterInspectors { get; }
```

Comment:
This is my suggested change.
```suggestion
        public ICollection<IParameterInspector> ParameterInspectors { get; }
```